src/esw/brushless.py

```diff
@@ -212,13 +218,108 @@ async def update(self) -> None:
             await self._connect()
 
 
-class DriveApp:
+class MotorsManager:
     BASESTATION_TO_ROVER_NODE_WATCHDOG_TIMEOUT_S = 1
-    DRIVE_NAMES = ["FrontLeft", "FrontRight", "MiddleLeft", "MiddleRight", "BackLeft", "BackRight"]
 
+    def __init__(self, motor_controller_info_by_name, transport, publish_topic: str):
+        self._motor_bridge_by_name = {}
+        self._motor_names = []
+        for name, info in motor_controller_info_by_name.items():
+            self._motor_names.append(name)
+            self._motor_bridge_by_name[name] = MoteusBridge(info["id"], transport)
+
+        self._last_updated_time = t.time()
+
+        self._motors_status_publisher = rospy.Publisher(publish_topic, MotorsStatus, queue_size=1)
+        self._motors_status = MotorsStatus(
+            name=[name for name in self._motor_names],
+            joint_states=JointState(
+                name=[name for name in self._motor_names],
+                position=[self._motor_bridge_by_name[name].moteus_data.position for name in self._motor_names],
+                velocity=[self._motor_bridge_by_name[name].moteus_data.velocity for name in self._motor_names],
+                effort=[self._motor_bridge_by_name[name].moteus_data.torque for name in self._motor_names],
+            ),
+            moteus_states=MoteusStateMsg(
+                name=[name for name in self._motor_names],
+                state=[self._motor_bridge_by_name[name].moteus_state.state for name in self._motor_names],
+                error=[self._motor_bridge_by_name[name].moteus_state.error_name for name in self._motor_names],
+            ),
+        )
+
+    def update_time(self) -> None:
+        """
+        Updates the last updated time
+        """
+        self._last_updated_time = t.time()
```

Comment:
Seems to me like it would make more sense to put this line in update_command_data() and get rid of this function call. Is there a reason this needs to be called before update_command_data()?

Reply:
yup that explanation makes sense i think